pkg/api/api.go

```diff
@@ -1210,11 +1269,37 @@ func (s *ScalewayAPI) CheckCredentials() error {
 		return err
 	}
 	if resp.StatusCode != 200 {
-		return fmt.Errorf("invalid credentials")
+		return fmt.Errorf("[%d] invalid credentials", resp.StatusCode)
 	}
 	return nil
 }
 
+// GetUUID returns the UUID
+func (s *ScalewayAPI) GetUUID() (string, error) {
```

Comment:
You should rename `GetUUID` and `UUID` to `GetUserID` and `UserID`, UUID is too much *universal* for this context